integration/class-enable-mastodon-apps.php

```diff
@@ -71,4 +83,58 @@ function ( $item ) {
 
 		return $followers;
 	}
+
+	/**
+	 * Resolve external accounts for Mastodon API
+	 *
+	 * @param Enable_Mastodon_Apps\Entity\Account $user_data The user data
+	 * @param string                              $user_id   The user id
+	 *
+	 * @return Enable_Mastodon_Apps\Entity\Account The filtered Account
+	 */
+	public static function api_account_external( $user_data, $user_id ) {
```

Comment:
what about renaming it to `api_account` and to extend also the internal accounts with `acct` and follower counts? Or should we do that in a different hook?

Reply:
Added that second hook in https://github.com/Automattic/wordpress-activitypub/pull/676